grafeas-charts/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: v1
-appVersion: "0.1"
+appVersion: "0.1.0"
 description: A Helm chart for Kubernetes
 name: grafeas-charts
-version: 0.1.0
+version: 0.1.1
+icon: https://raw.githubusercontent.com/appvia/grafeas/master/logo/grafeas-logo-128.png
```

Comment:
This should link to the grafeas/grafeas, right?

Reply:
Yep will update.

Reply:
Updated.